### PR TITLE
 Changes to provide Summary data output of additional block variables

### PR DESCRIPTION
**Pull request by jalvestad**
Provide output data for:

    BOSAT
    BWKR
    BOKR
    BKRO
    BGKR
    BKRG
    BKRW
    BWPC
    BGPC
    BVWAT
    BWVIS
    BVGAS
    BGVIS
    BVOIL
    BOVIS

This pull request is needed in order to output Summary data, see pull request OPM/opm-common#724

### CODE REVIEW

File: ebos/ecloutputblackoilmodule.hh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```
Comment (andlaus) [inline ebos/ecloutputblackoilmodule.hh]: this change seems unintentional.

File: ebos/ecloutputblackoilmodule.hh [hunk 2]
```diff
@@ -307,7 +307,7 @@ public:
         }
 
         if (FluidSystem::phaseIsActive(waterPhaseIdx) && rstKeywords["KRW"] > 0) {
-            rstKeywords["KRW"] = 0;
+	    rstKeywords["KRW"] = 0;
```
Comment (andlaus) [inline ebos/ecloutputblackoilmodule.hh]: tabulator instead of spaces? That hunk also seems to be an artifact.

**Comment by bska** [comment]
jenkins build this ignore_extra please

**Comment by bska** [comment]
jenkins build this opm-common=724 ignore_extra please

File: ebos/ecloutputblackoilmodule.hh (hunk 1 above)
Comment (jalvestad) [reply ebos/ecloutputblackoilmodule.hh]: Yes - this was unintentional (sorry)- has now been removed!

File: ebos/ecloutputblackoilmodule.hh (hunk 2 above)
Comment (jalvestad) [reply ebos/ecloutputblackoilmodule.hh]: This was also unintentional and has been removed.

**Comment by jalvestad** [comment]
The pull request is finished and ready to merge unless there are comments from the maintainers. Due to the changes made, i.e. summary variables added, update data needs to be performed.

**Comment by totto82** [comment]
Any reasons why this and its fried in opm-common is not merge? 
Jenkins is asked to "ignore_extra" does that mean that we need a update_data before this can be merged?

**Comment by joakim-hove** [comment]
> Any reasons why this and its fried in opm-common is not merge? 

The one in opm-common has not had a green Jenkins, but apart from that it should be ready to go in.

**Comment by bska** [comment]
> The one in opm-common (OPM/opm-common#724) has not had a green Jenkins

And will require updating the reference data before it gets a green CI result.  The PR could probably do with a rebase too.

**Comment by jalvestad** [comment]
This pull request has also been rebased now, and is hopefully ready to be merged.

**Comment by bska** [comment]
I'm merging this now along with OPM/opm-common#724 and OPM/opm-tests#116 to bring in the new block quantities.  Thanks a lot for your efforts!

**Pull request merged**